lib/manageiq/appliance_console/database_admin.rb

```diff
@@ -19,6 +20,11 @@ class DatabaseAdmin < HighLine
         Example: 'mydomain.com/user'
       PROMPT
 
+      ACCESS_KEY_PROMPT = <<-PROMPT.strip_heredoc.chomp
```

Comment:
S3 is the only method that will use this, right? If that's the case, can this string live in the `#ask_s3_file_options` method?

Reply:
Will do...